CHANGELOG.md

```diff
@@ -1,3 +1,91 @@
+## streamlink 0.2.0 (12-16-2016)
+
+Release 0.2.0 of Streamlink!
+
+We've done numerous changes to plugins as well as fixed quite a few which were originally failing. Amung these changes are updated docs as well as general UI/UX cleaning with console output.
```

Comment:
Yeah this used to happen for me because I had two different tags (one machine versus another) in my git config.